agent/tool-scripts/pprof

```diff
@@ -146,7 +160,15 @@ ose_pprof() {
 
 collect_data() {
     while true; do
-            $tool_bin tool $tool -text -seconds=$interval -output $PPROF_TMPDIR/$(date +%Y-%m-%d:%H:%M:%S)_.txt http://localhost:6060/debug/pprof/profile
+        if [ $(systemctl list-unit-files | grep atomic-openshift-master.service | awk {'print $2'}) == "masked" ]; then
+            $tool_bin tool $tool -text -seconds=$interval -output $PPROF_TMPDIR/$(date +%Y-%m-%d:%H:%M:%S)_controller.txt http://localhost:6060/debug/pprof/profile 
```

Comment:
@portante Yes, they run serially,   the first one (profile) samples for $interval seconds and then exits.   The second one (heap) runs and returns immediately  which cycles the loop and starts the next sample.  I'm not a big fan of this overloading of interval + the --seconds parameter does not do anything on heap.  I will follow up with another PR to clean this up.   Let me know if you have comments on doing it better.